5.0/en/0x14-V6-Cryptography.md

```diff
@@ -2,62 +2,318 @@
+- Implementing robust cryptographic systems that fail securely, adapt to evolving threats, and are future-proof.
+- Utilizing cryptographic mechanisms that are both secure and aligned with industry best practices.
+- Maintaining a secure cryptographic key management system with appropriate access controls and auditing.
+- Regularly evaluating the cryptographic landscape to assess new risks and adapt algorithms accordingly.
+- Discovering and managing cryptographic use cases throughout the application's lifecycle to ensure that all cryptographic assets are accounted for and secured.
 
-## V6.1 Data Classification
+## V6.1 Data Classification and Cryptographic Inventory
 
-The most important asset is the data processed, stored or transmitted by an application. Always perform a privacy impact assessment to classify the data protection needs of any stored data correctly.
+The most important asset is the data processed, stored, or transmitted by an application. Always perform a privacy impact assessment to classify the data protection needs of any stored data correctly. In addition, ensure that all cryptographic assets, such as algorithms, keys, and certificates, are regularly discovered, inventoried, and assessed.
 
-| # | Description | L1 | L2 | L3 | CWE |
-| :---: | :--- | :---: | :---: | :---: | :---: |
-| **6.1.1** | Verify that regulated private data is stored encrypted while at rest, such as Personally Identifiable Information (PII), sensitive personal information, or data assessed likely to be subject to EU's GDPR. | | ✓ | ✓ | 311 |
-| **6.1.2** | Verify that regulated health data is stored encrypted while at rest, such as medical records, medical device details, or de-anonymized research records. | | ✓ | ✓ | 311 |
-| **6.1.3** | Verify that regulated financial data is stored encrypted while at rest, such as financial accounts, defaults or credit history, tax records, pay history, beneficiaries, or de-anonymized market or research records. | | ✓ | ✓ | 311 |
+|     #     | Description                                                                                                                                                                                                                                                                                         | L1  | L2  | L3  | CWE |
+| :-------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: | :-: | :-: | :-: |
+| **6.1.1** | Verify that regulated data, such as private, health, and financial, is stored encrypted while at rest with at least 112-bit encryption, ideally 128-bit, such as Personally Identifiable Information (PII), sensitive personal information, or data assessed likely to be subject to the EU's GDPR. |     |  ✓  |  ✓  | 326 |
```

Comment:
"such as" is used twice in this sentence but is used for the same thing. It should be read as:

> Verify that regulated data, such as private, health, financial,  Personally Identifiable Information (PII), sensitive personal information, or data assessed likely to be subject to the EU's GDPR, is stored encrypted while at rest with at least 112-bit encryption, ideally 128-bit.

but does not parse like so for me in the current formulation.

Reply:
Is it clear for everyone what "encrypted while at rest means"? For example, for databases should I encrypt the data at the column level (by the application itself) ? or is filesystem encryption enough ?